Tests/BootCamp.Chapter.Tests/Expected/Kwiki Mart.csv

```diff
@@ -0,0 +1,4 @@
+﻿City, Street, Item, DateTime, Price
+Springfield, HoloPapap 2, Sausage, 2020-01-05T20:00:00Z, "8,00 €"
+Springfield, HoloPapap 2, Bannana, 2020-01-05T22:00:00Z, "10,00 €"
+Springfield, HoloPapap 2, Bannana, 2020-01-05T22:00:00Z, "15,50 €"
```

Comment:
Currently format change. 

File renamed, because the expectation is the file name matches the shop name.